app/views/admin/facilities/_form.html.erb

```diff
@@ -18,7 +18,7 @@
           required: true %>
       <small class="form-text text-muted mb-3">
         If you don't see a block in the list,
```

Comment:
@timcheadle this should say "If you don't see a ${block} in the list". Block should be localized too :)

Reply:
Fixing.